File: FadingAnimations/app/src/main/java/com/example/muskangoyal/fadinganimations/MainActivity.java
package com.example.muskangoyal.fadinganimations;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    ImageView jerry,pikachu;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        jerry= findViewById(R.id.jerry);
        pikachu=findViewById(R.id.pikachu);


        jerry.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
               // jerry.animate().alpha(0f).setDuration(2000);
                //pikachu.animate().alpha(1f).setDuration(2000);
                //jerry.animate().translationXBy(1000f).setDuration(2000);
                jerry.animate().rotation(360).setDuration(1000);
            }
        });
    }
}

File: KotlinTodoList/app/src/main/java/com/example/muskangoyal/kotlintodolist/MainActivity.kt
package com.example.muskangoyal.kotlintodolist

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import com.example.muskangoyal.kotlintodolist.Adapter.TodoAdapter
import com.example.muskangoyal.kotlintodolist.models.Todo
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    val todos: ArrayList<Todo> = ArrayList();

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        todos.add(Todo("First task",false))

        rvtodo.layoutManager= LinearLayoutManager(this)
        val todoAdapter : TodoAdapter = TodoAdapter(todos);
        rvtodo.adapter= todoAdapter;

        btn.setOnClickListener({
            val newtodo =et1.text.toString()
            todos.add(Todo(newtodo,false))
            todoAdapter.notifyDataSetChanged();
        }
        )
    }
}
